frontend/packages/dev-console/src/components/health-checks/HealthChecks.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { PageHeading } from '@console/internal/components/utils';
+import FormSection from '../import/section/FormSection';
+import HealthCheckProbe from './HealthCheckProbe';
+import { HealthChecksProbeType } from './health-checks-types';
+
+interface HealthChecksProps {
+  pageTitle?: string;
+}
+
+const renderProbes = () => {
+  return (
+    <>
+      <HealthCheckProbe probe={HealthChecksProbeType.ReadinessProbe} />
+
+      <HealthCheckProbe probe={HealthChecksProbeType.LivenessProbe} />
+
+      <HealthCheckProbe probe={HealthChecksProbeType.StartupProbe} />
+    </>
+  );
+};
+
+const HealthChecks: React.FC<HealthChecksProps> = ({ pageTitle }) => {
+  return pageTitle ? (
+    <>
+      <PageHeading title={pageTitle} />
+      {renderProbes()}
```

Comment:
this method does not depend on any conditions or variables. It would be a better idea to just call the three components here itself.

Reply:
`renderProbes` has been used at line 30 as well

Reply:
Why do you need a separate way of defining page title anyways? I think it should always be a `HealthChecksSection` component which is a form section. The form using this section should be responsible of rendering it properly with page title and what not.

Reply:
@rohitkrai03 Yes, we are always going to use this for health checks form section. So, here are two scenarios. For add flow, we need form section title and for the health check page, we don't need form section title as we already have page heading.

Reply:
Right, `FormSection` has `title` as an optional prop. Just make use of that. If a title is sent from the parent it will render it if not then no title will be rendered.